build/package/chocolatey/atlascli.nuspec

```diff
@@ -2,12 +2,15 @@
 <!-- Do not remove this test for UTF-8: if “Ω” doesn’t appear as greek uppercase omega letter enclosed in quotation marks, you should use an editor that supports UTF-8, not this one. -->
 <package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
   <metadata>
-    <id>atlascli</id>
+    <id>mongodb-atlas</id>

```

Comment:
this is the main change, all other changes were needed given we've had some instability on nupkg generation